src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -80,6 +87,36 @@ const DropdownWithinOverflownContainer: StoryFn<DropdownProps> = (args: Dropdown
     );
 };
 
+const DropdownWithBackgroundTemplate: StoryFn<DropdownProps> = (args: DropdownProps) => {
+    const [active, setActive] = useState(args.activeItemId);
+    const [active2, setActive2] = useState(args.activeItemId);
+    useEffect(() => setActive(args.activeItemId), [args.activeItemId]);
+    useEffect(() => setActive2(args.activeItemId), [args.activeItemId]);
+
+    return (
+        <div className="tw-p-8 tw-bg-black-5 tw-flex tw-gap-2">
+            <div className="tw-max-w-[350px]">
```

Comment:
For the story here, I would remove the default emphasis dropdown and have the week one a bit larger than 350px

Reply:
Makes sense, removed.